app/assets/javascripts/admin/addon/controllers/admin-emojis-new.js

```diff
@@ -0,0 +1,27 @@
+import Controller from "@ember/controller";
+import EmberObject, { action, computed } from "@ember/object";
+import { service } from "@ember/service";
+
+const ALL_FILTER = "all";
+
+export default class AdminEmojisNewController extends Controller {
+  @service router;
+
+  @computed("model")
+  get emojiGroups() {
+    return this.model.mapBy("group").uniq();
+  }
+
+  @computed("emojiGroups.[]")
+  get sortingGroups() {
+    return [ALL_FILTER].concat(this.emojiGroups);
+  }
+
+  @action
+  emojiUploaded(emoji, group) {
+    emoji.url += "?t=" + new Date().getTime();
+    emoji.group = group;
```

Comment:
This also needs to set `created_by`, otherwise that column is blank after you make the emoji

Reply:
Wow. What a catch! 👨‍🍳👌